woocommerce/woocommerce-ios
# [CPP Onboarding] Implement view for plugin not activated

Part of #4611 

This adds a specific onboarding error screen for the case when WCPay is installed but not activated.

## To test

You should have a store with an address in the US and with the WooCommerce Payments installed.

1. From wp-admin, deactivate WooCommerce Payments from Plugins > Installed Plugins.
2. Go to Settings > In-Person Payments. The app should show a screen prompting you to activate the extension.
3. From wp-admin, activate WooCommerce Payments from Plugins > Installed Plugins.
3. Tap on Refresh After Activating.
4. The app should show a generic error (since payments aren't fully set up yet).

| Extension not activated | After tapping Refresh |
| ---- | ---- |
| <img src="https://user-images.githubusercontent.com/2722505/128708219-a659344d-287f-4120-84b4-7b18e12c89ca.png" width="350"/> | <img src="https://user-images.githubusercontent.com/2722505/128708221-e015df30-5a78-4b60-9999-7c1361d2a2f1.png" width="350"/> |


Update release notes:
